### PR TITLE
feat: Add new TypingDots UI component

### PR DESCRIPTION
**Pull request by liamcho**
Fixes: [UIKIT-4563](https://sendbird.atlassian.net/browse/UIKIT-4563)

https://www.webmobilefirst.com/en/screencasts/m8A0hyjGJS/

figma design: https://www.figma.com/file/SVbXU00FhjztekD8AiVukK/UIKit_Work-file_React?node-id=1869%3A800&mode=dev

[UIKIT-4563]: https://sendbird.atlassian.net/browse/UIKIT-4563?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/ui/TypingIndicatorMessage/typingDots.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
+@import '../../styles/variables';
+
+$dot-diameter-default: 8px;
+$duration: 1.4s;
+$blinked-scale: 1.2;
```
Comment (bang9) [inline src/ui/TypingIndicatorMessage/typingDots.scss:5]: image
It seems like the animation scale is set to 150% in Figma.
However, when I apply 1.5x, it looks too large 🤔 We should show the result to Gabie and double-check.

Comment (liamcho) [reply src/ui/TypingIndicatorMessage/typingDots.scss:5]: yeah i will do that. That is why i set to 1.2

Comment (liamcho) [reply src/ui/TypingIndicatorMessage/typingDots.scss:5]: Gabie says 1.2 is good.

**Review by bang9 — APPROVED** [review]
lgtm!
remember to remove the test code before merging into the main

**Review by HoonBaek — APPROVED** [review]

**Comment by liamcho** [comment]
@HoonBaek @bang9 Ill merge this PR but i will be making another PR for updating this UI with new design that is to come (currently still in design phase).

New design in process:
Screenshot 2023-11-07 at 1 39 52 PM

**Pull request merged**